src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -16,6 +16,7 @@ namespace NuGet.Services.Metadata.Catalog.Persistence
     public class AzureStorage : Storage
     {
         private readonly CloudBlobDirectory _directory;
+        private readonly int MaxCountOfBlobSnapshots = 30;
```

Comment:
Also, this code should be reused (@agr moved it to a shared repo), and not for all usages we will require snapshots.